src/main/java/com/teksystems/TekcampExerciseSpring/Services/UserService.java

```diff
@@ -0,0 +1,21 @@
+package com.teksystems.TekcampExerciseSpring.Services;
+
+
+
+import java.util.Optional;
+
+import com.teksystems.TekcampExerciseSpring.Models.UserEntity;
+import com.teksystems.TekcampExerciseSpring.Models.Response.UserResponse;
+import com.teksystems.TekcampExerciseSpring.Shared.Dto.UserDto;
+
+
+public interface UserService {
+	UserDto createUser(UserDto userDto);
+	Iterable<UserEntity> displayUsers(int page, int limit);
+	void deleteUser(Long id);
+	Optional<UserEntity> findById(Long id);
+	Optional<UserEntity> getUserByEmail(String email);
+	UserResponse updateUser(UserEntity userEntity,Long id);
```

Comment:
Good way to check if you're using DTOs where you need to is to check the Service interface. These should all be Dto(s) or void